test/acquisition/multi_objective/test_monte_carlo.py

```diff
@@ -641,6 +641,19 @@ def test_q_noisy_expected_hypervolume_improvement(self):
             X = torch.zeros(1, 1, **tkwargs)
             # basic test
             sampler = IIDNormalSampler(sample_shape=torch.Size([1]))
+
+            # test error is raised if m == 1
+            if m == 1:
+                with self.assertRaises(ValueError):
+                    acqf = qNoisyExpectedHypervolumeImprovement(
+                        model=mm,
+                        ref_point=ref_point,
+                        X_baseline=X_baseline,
+                        sampler=sampler,
+                        cache_root=False,
+                    )
+                continue
```

Comment:
```suggestion
```

Is this needed?

Reply:
I am added `continue` to skip the test afterwards. Or do we need to run full test cases for`m=1`?

Reply:
Oh yeah, that makes sense :)